en/mkdocs.yml

```diff
@@ -727,6 +727,7 @@ nav:
           - "Migrating to 5.11.0" : setup/migrating-to-5110.md
           - 'Migrating Data Publishers': setup/migrating-data-publishers.md
           - 'Migrating Custom Scope Validators': setup/migrating-custom-scope-validators.md
+      - 'WSO2 Identity Server Feature Deprecation': setup/iam-feature-maintenance-policy.md
```

Comment:
shall we rename the md file name too to be consistent with other files?

Reply:
Fixed https://github.com/wso2/docs-is/pull/2205/commits/1d65b1f4f8816b0f5d389b3ece009572717ae456